README.md

```diff
@@ -57,52 +52,20 @@ class {'mongodb::server':
 }
 ```
 
-For Red Hat family systems, the client can be installed in a similar fashion:
+To install client with default options run `include mongodb::client`
 
-```puppet
-class {'mongodb::client':}
-```
-
-Note that for Debian/Ubuntu family systems the client is installed with the
-server. Using the client class will by default install the server.
-
-If one plans to configure sharding for a Mongo deployment, the module offer
-the `mongos` installation. `mongos` can be installed the following way :
-
-```puppet
-class {'mongodb::mongos' :
-  configdb => ['configsvr1.example.com:27018'],
-}
-```
-
-Although most distros come with a prepacked MongoDB server, you may prefer to
-use a more recent version. To install MongoDB from the community repository:
+To override the default mongodb repo version you need the following:
 
 ```puppet
 class {'mongodb::globals':
-  manage_package_repo => true,
-  version             => '3.6',
-}
--> class {'mongodb::client': }
--> class {'mongodb::server': }
-```
-
-If you don't want to use the MongoDB software repository or the OS packages,
-you can point the module to a custom one.
-To install MongoDB from a custom repository:
```

Comment:
I removed this part as managing the repo in a custom location is hard with apt keys.
Even though it is still possible to do this with yum based repos I would prefer to remove that code from the module later.
In my opinion this should be managed in a local profile class.

Reply:
I wouldn't put to much effort in this file.  This will be replaces with the reference.md, as soon as we moves the docu inside the manifests.  His commit was an attempt of me https://github.com/voxpupuli/puppet-mongodb/pull/677/commits/07eac436d3022ba189166a84901721ba50e6d479

Reply:
Thats why I didn't bother changing anything in the reference section.

Reply:
I prefere having some small setup and usage examples in readme even with reference.md

Reply:
thats fine with me ...